private/buf/cmd/buf/command/registry/registrylogin/registrylogin.go

```diff
@@ -44,8 +44,8 @@ func NewCommand(
 		// Not documenting the first arg (remote) as this is just for testing for now.
 		// TODO: Update when we have self-hosted.
 		Use:   name,
-		Short: `Login to the Buf Schema Registry.`,
-		Long:  fmt.Sprintf(`This will prompt for your BSR username and a BSR token, and update your %s file with these credentials.`, netrc.Filename),
+		Short: `Log in to the Buf Schema Registry.`,
```

Comment:
This one always forces me to look up the correct form 🤦

Reply:
I feel your pain 👊🏼